README.md

```diff
@@ -157,11 +157,12 @@ They require JetPack 4.2 and above, and [@dusty-nv](https://github.com/dusty-nv)
 If you are installing from source, you will need:
 - Python 3.7 or later (for Linux, Python 3.7.6+ or 3.8.1+ is needed)
 - A C++17 compatible compiler, such as clang
+- If using cpp/gcc on ubuntu consider switching from apt package `g++-11` to `g++-9` [to avoid reported compilation issues](https://github.com/pytorch/pytorch/issues/91328)
 
 We highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment. You will get a high-quality BLAS library (MKL) and you get controlled dependency versions regardless of your Linux distro.
 
 If you want to compile with CUDA support, install the following (note that CUDA is not supported on macOS)
-- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.0 or above
+- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.0 or above [**Note: CUDA 12 is not currently supported**](https://github.com/pytorch/pytorch/issues/91122)
```

Comment:
I think maybe we should just link to this page: https://pytorch.org/get-started/locally/ as it has most accurate matrix of what's supported.

Reply:
Note:  this is section we're editing is build from source and... https://pytorch.org/get-started/locally/#linux-from-source points here. Bit of a loop if we point there :-)

Reply:
@svekars friendly ping :-)

Reply:
updated...

Reply:
@svekars wdyt :-)